libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -57,17 +57,18 @@ type
     heartbeatRunning: bool
     heartbeatLock: AsyncLock                   # heartbeat lock to prevent two consecutive concurrent heartbeats
 
-declareGauge(libp2p_gossipsub_peers_per_topic_mesh,
-  "gossipsub peers per topic in mesh",
-  labels = ["topic"])
+when defined(libp2p_expensive_metrics):
```

Comment:
these should be added back, they're not expensive - ie the topic list is fairly limited (70 items)

Reply:
yes, it wouldn't hurt, but we can get them back with the define, so might not be a big issue?

Reply:
> the topic list is fairly limited (70 items)

How many times can the fork version change in 15 days?